ubiquibot-config-default.json

```diff
@@ -8,6 +8,7 @@
   "disable-analytics": false,
   "comment-incentives": false,
   "register-wallet-with-verification": false,
+  "stale-bounty-time": 42060,
```

Comment:
Perhaps we should consider changing this file to `ubiquibot-config-default.ts` because I would love to be able to include comments. We cant rely on `fs.readFile` due to limitations with Netlify, so we can not do `.jsonc` (I already tried on a pull request earlier this month.

Reply:
I'll leave this for now until it's decided then

Reply:
@rndquu what are your thoughts for this, can we close the issue and decide this later or just leave it open for now?